cypress/pages/index.jsx

```diff
@@ -0,0 +1,64 @@
+import selectors from "../selectors/Login";
+
+class LoginPage {
+  constructor() {
+    this.selectors = selectors;
```

Comment:
you don't need to set the this.selectors = selectors within the class since it's already being imported from /selectors/Login. You can do cy.get(selectors.loginEmailInput) instead